### PR TITLE
#897: Results Choropleth: Updated Legend Text Sizing and Legend Sizing

### PR DESCRIPTION
**Pull request by sarahsmo**
Text size of the Results legend was too small. I increased the font size. I also increased the width of the Legend to better fit the font increase.

### CODE REVIEW

**Comment by jonfroehlich** [comment]
Can i see before and after screenshots?

Congrats on your first commit.

**Comment by sarahsmo** [comment]
Before:
before

**Comment by sarahsmo** [comment]
After:
after

File: build.sbt [hunk 1]
```diff
@@ -83,3 +83,6 @@ ScalariformKeys.preferences := ScalariformKeys.preferences.value
   .setPreference(PreserveDanglingCloseParenthesis, true)
 
 fork in run := true
+
+
+fork in run := true
```
Comment (misaugstad) [inline build.sbt]: I am assuming these edits to the build file were unintentional? If so, please remove them! And before committing anything, always do a `git status` and make sure the files that are marked as modified are what you expect.

Comment (sarahsmo) [reply build.sbt]: Okay should be good to go now

Comment (sarahsmo) [reply build.sbt]: @misaugstad let me know if there is anything else :)

**Comment by misaugstad** [comment]
looks good!

**Comment by manaswis** [comment]
@sarahsmo It comes on 4 lines for me and too close to the borders - 
image

**Comment by manaswis** [comment]
Hmm, weird. The new PR merge with the position update fixed the spacing issues.

image

**Pull request merged**